src/Pact/Gas/Table.hs

```diff
@@ -57,6 +57,8 @@ defaultGasTable =
   [
    -- General native functions
     ("at",                   const 2)
+   ,("base64-decode",        const 1)
```

Comment:
Maybe throw a TODO on these so we know to swing back to them next time we do a gas report.

Reply:
Added